src/containers/leaderboard/Leaderboard.js

```diff
@@ -138,14 +140,10 @@ class Leaderboard extends React.Component {
                     {this.renderRoundsBlocks(count)}
                   </div>
                 </div>
-                {this.renderRound(roundInfo)}
-                <div className="ratings-tabs">
-                  <div className="ratings-tab ratings-traders active">
-                    <div className="ratings-table-wrap js-table-wrapper">
-                      {this.state.selectedRound === 0 ? this.renderGlobalBoard(sortedData) : this.renderRoundBoard(sortedData, roundInfo)}
-                    </div>
-                  </div>
-                </div>
+                {isSelectedRoundExist ?
+                  this.renderRound(roundInfo) && this.renderMainBoard(sortedData, roundInfo) :
```

Comment:
Call to `renderRound` allocates new react components if `roundInfo` is truly. This components are never been in use. Using of ternal operator leads to use this tricky construction. So whole construction should be moved into separated method `renderBoard` which has simple and strict codeflow.

What should be done.

This code:

```javascript
isSelectedRoundExist ?
                  this.renderRound(roundInfo) && this.renderMainBoard(sortedData, roundInfo) :
                  this.renderMissingRoundNotice()
```

 should be replaced with:
```javascript
this.renderBoard(sortedData, roundInfo, isSelectedRoundExists)
```

Add function `renderBoard`:
```javascript
renderBoard(sortedData, roundInfo, isSelectedRoundExists) {
    if (! isSelectedRoundExists) {
          return this.renderMissingRoundNotice();
    }
    else if (roundInfo) {
          return this.renderMainBoard(sortedData, roundInfo)
    }
    else {
         return null;
    }
}
```